tests/tests_register_unregister.yml

```diff
@@ -3,9 +3,7 @@
 - name: Basic register/unregister
   hosts: all
   gather_facts: true
-  become: true
-  tags:
-    - tests::slow
```

Comment:
this tag flags that the test is slow, which is the case when running it as default without an external candlepin

Reply:
Sorry Pino! I bring back those lines